examples/experimental/bezier/package.json

```diff
@@ -4,8 +4,8 @@
     "start": "webpack-dev-server --progress --hot --open"
   },
   "dependencies": {
-    "deck.gl": ">=5.2.0-alpha.8",
-    "deck.gl-layers": "0.1.0-beta.2",
+    "deck.gl": "^5.2.0",
+    "@deck.gl/experimental-layers": "^5.2.0",
```

Comment:
This is a different module. Doesn't this require a source code change?

Reply:
No, the source is already importing from `@deck.gl/experimental-layers`.